darts/models/forecasting/ensemble_model.py

```diff
@@ -377,6 +386,117 @@ def _predictions_reduction(
+        pkl_kwargs
+            Keyword arguments passed to `pickle.dump()`
+        """
+
+        if path is None:
+            # default path
+            path = self._default_save_path() + ".pkl"
+
+        if isinstance(path, (str, os.PathLike)):
+            # save the whole object using pickle
+            with open(path, "wb") as handle:
+                pickle.dump(obj=self, file=handle, **pkl_kwargs)
+        elif isinstance(path, io.BufferedWriter):
+            # save the whole object using pickle
+            pickle.dump(obj=self, file=path, **pkl_kwargs)
+        else:
+            raise_log(
+                ValueError(
+                    "Argument 'path' has to be either 'str' or 'PathLike' (for a filepath) "
+                    f"or 'BufferedWriter' (for an already opened file), but was '{path.__class__}'."
+                ),
+                logger=logger,
+            )
+
+        for i, m in enumerate(self.forecasting_models):
+            if TORCH_AVAILABLE and issubclass(type(m), TorchForecastingModel):
+                path_pt_ckpt = f"{path}.{type(m).__name__}_{i}.pt"
+                m.save(path=path_pt_ckpt)
+
+    @staticmethod
+    def load(path: Union[str, os.PathLike, BinaryIO]) -> "EnsembleModel":
+        """
+        Loads the ensemble model from a given path or file handle.
+
+        Parameters
+        ----------
+        path
+            Path or file handle from which to load the ensemble model.
+        """
+
+        if isinstance(path, (str, os.PathLike)):
+            raise_if_not(
+                os.path.exists(path),
+                f"The file {path} doesn't exist",
+                logger,
+            )
+
+            with open(path, "rb") as handle:
+                model = pickle.load(file=handle)
+        elif isinstance(path, io.BufferedReader):
+            model = pickle.load(file=path)
+        else:
+            raise_log(
+                ValueError(
+                    "Argument 'path' has to be either 'str' or 'PathLike' (for a filepath) "
+                    f"or 'BufferedReader' (for an already opened file), but was '{path.__class__}'."
+                ),
+                logger=logger,
+            )
+
+        for i, m in enumerate(model.forecasting_models):
+            if issubclass(type(m), TorchForecastingModel):
+                # if a checkpoint was saved, we also load the PyTorch LightningModule from checkpoint
+                path_ptl_ckpt = f"{path}.{type(m).__name__}_{i}.pt.ckpt"
+                if os.path.exists(path_ptl_ckpt):
+                    m.model = m._load_from_checkpoint(path_ptl_ckpt)
+                else:
+                    m._fit_called = False
+                    logger.warning(
+                        f"Model was loaded without weights since no PyTorch LightningModule checkpoint ('.ckpt') could be "  # noqa: E501
+                        f"found at {path_ptl_ckpt}. Please call `fit()` before calling `predict()`."
+                    )
```

Comment:
the reason it failed is that we need to replace `m` itself, not `m.model` 

```suggestion
        for i, m in enumerate(model.forecasting_models):
            if TORCH_AVAILABLE and issubclass(type(m), TorchForecastingModel):
                path_tfm = f"{path}.{type(m).__name__}_{i}.pt"
                model.forecasting_models[i] = TorchForecastingModel.load(path_tfm)
```